src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        Lottery ball2 = new Lottery();
+        Lottery ball3 = new Lottery();
```

Comment:
Why do we need here few instances of type `Lottery`?
Also, for variable with type `Lottery` name ball1 can be confusing (something like lottery would be better)